documents/invoice/handler.go

```diff
@@ -34,6 +34,7 @@ func GRPCHandler() (clientinvoicepb.DocumentServiceServer, error) {
 
 // Create handles the creation of the invoices and anchoring the documents on chain
 func (h *grpcHandler) Create(ctx context.Context, req *clientinvoicepb.InvoiceCreatePayload) (*clientinvoicepb.InvoiceResponse, error) {
+	apiLog.Infof("Create request %v", req)
```

Comment:
it's fine for now, right now not enough logging is the problem. we can think of removing unnecessary stuff later.

Reply:
I agree there is not enough logging but these changes will be too much noise since you are logging entire requests. Imagine how much unneeded logging that would be for invoice data or PO data.

Reply:
I will make these debug statements

Reply:
Are those messages clear enough in the logs? "Update request" / "Get request" doesn't say much what it is when looked at in the log output.

Reply:
the logs seems clear because the logger is specific to the service and module.